apps/mobile/src/components/action-bar/action-bar.tsx

```diff
@@ -13,7 +14,7 @@ import { useTheme } from '@shopify/restyle';
 import { BlurView, Box, Theme } from '@leather.io/ui/native';
 
 export const ACTION_BAR_HEIGHT = 70;
-export const ACTION_BAR_BOTTOM_OFFSET = 40;
+export const ACTION_BAR_BOTTOM_OFFSET = 10;
```

Comment:
Thank you! This had been annoying me for so long haha

Reply:
Same. The plan is the improve this a bit further in the future: [LEA-2369: Adjust the positioning of the action bar](https://linear.app/leather-io/issue/LEA-2369/adjust-the-positioning-of-the-action-bar)